### PR TITLE
[testcontainers] Testcontainers languages reference

### PR DESCRIPTION
**Pull request by refaeli-io**
### Description
Add some references to Testcontainers languages implementations with selenium: 
[dotnet](https://github.com/testcontainers/testcontainers-dotnet)
[java](https://github.com/testcontainers/testcontainers-java)
[python](https://github.com/testcontainers/testcontainers-python)

[main website](https://testcontainers.com/)

### Types of changes

- [x] Change to the site (I have double-checked the Netlify deployment, and my changes look good)
- [ ] Code example added (and I also added the example to all translated languages)
- [ ] Improved translation
- [ ] Added new translation (and I also added a notice to each document missing translation)

### Checklist

- [x] I have read the [**contributing**](https://selenium.dev/documentation/en/contributing/) document.
- [x] I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.

### CODE REVIEW

**Review by diemol — COMMENTED** [review]
@ShaharM7 thank you for this PR!

I think this needs a new section, because TestContainers is neither a language binding nor a framework, I believe it is a tool that supports and enables testing. 

Maybe the new section can be called "Tools".

Also, we should not link only to the .NET flavor, but rather to all the relevant ones from TestContainers (like .NET, Java, Python...)

@ShaharM7 are you part of the TestContainers team? Ideally we want to feature them with their approval.

**Comment by eddumelendez** [comment]
> Also, we should not link only to the .NET flavor, but rather to all the relevant ones from TestContainers (like .NET, Java, Python...)

Totally agree with this.

The new section called `Tools` makes totally sense for me.

@diemol I am part of the Testcontainers team. So, once the changes are done, I can approve it :)

File: website_and_docs/content/ecosystem/_index.html
Comment (eddumelendez) [inline website_and_docs/content/ecosystem/_index.html]: Link should be https://testcontainers.com/

Comment (refaeli-io) [reply website_and_docs/content/ecosystem/_index.html]: And I mention all the relevant languages: .NET, Java, Python?

Comment (eddumelendez) [reply website_and_docs/content/ecosystem/_index.html]: Testcontainers provides Java, .Net, Go, Python and mentioned in the original issue currently there is only modules in Java and Python. .Net is still WIP

**Comment by refaeli-io** [comment]
@diemol 
I am not part of the team, but maybe in the future @eddumelendez hmmm :)
Also, I agree this is a tool - an integration testing tool, and give @eddumelendez to approve he is part of the team but give me also some credit on that dotnet implemetion 😄

File: website_and_docs/content/ecosystem/_index.html
Comment (eddumelendez) [inline website_and_docs/content/ecosystem/_index.html]: Once the PR in Testcontainers for .NET is merged, .NET can be added instead of C#

```suggestion
        <td>Java, Python</td>
```

File: website_and_docs/content/ecosystem/_index.html
Comment (eddumelendez) [inline website_and_docs/content/ecosystem/_index.html]: @kiview is ok to have Testcontainers as an author of Testcontainers project? 🤔

**Review by eddumelendez — COMMENTED** [review]
Just left a couple of comments. Once those are fixed/confirmed I'll approve it.

File: website_and_docs/content/ecosystem/_index.html
Comment (kiview) [reply website_and_docs/content/ecosystem/_index.html]: I think the correct way to document this would be referencing the `Webdriver Container` module, with the `Testcontainers Community` (modelled after `Facebook & Community`) project as the author. It is probably not correct in the strict copyright sense, but connecting it to a single person does not make much sense in the context of this ecosystem page.

Comment (kiview) [inline website_and_docs/content/ecosystem/_index.html]:
```suggestion
        <td>Testcontainers Community</td>
```

File: website_and_docs/content/ecosystem/_index.html
Comment (refaeli-io) [reply website_and_docs/content/ecosystem/_index.html]: It will be merged soon 
I Keep it open

Comment (eddumelendez) [reply website_and_docs/content/ecosystem/_index.html]: I think it will be faster to merge this after applying the suggestions instead of waiting until it's not only merge but release it in testcontainers-dotnet. It will also easy to add `.Net` later in this section.

**Review by eddumelendez — APPROVED** [review]
LGTM

@diemol 👍🏽 from my side

Thanks @ShaharM7 !

**Comment by refaeli-io** [comment]
If LGTM approves and merge 
I do it for Testcontainer because I believe that project 👍

**Comment by luisfcorreia** [comment]
While not a translation issue I can look at, this change does indeed LGTM

**Comment by ShaharM7** [comment]
@eddumelendez 
The issue [is closed 🎉](https://github.com/testcontainers/testcontainers-dotnet/pull/780#issuecomment-1464869883) [ and merged 
](https://github.com/testcontainers/testcontainers-dotnet/pull/780#issuecomment-1464869883)
So I added .NET to the languages

**Comment by refaeli-io** [comment]
@eddumelendez 
This issue [is closed 🎉](https://github.com/testcontainers/testcontainers-dotnet/issues/762#event-8723741845) [ and merged 
](https://github.com/testcontainers/testcontainers-dotnet/pull/780#issuecomment-1464869883)
So I added .NET to the languages 😃

**Review by diemol — APPROVED** [review]
Thank you everyone!

**Pull request merged**
